src/components/NewMessage/NewMessage.vue

```diff
@@ -803,71 +801,57 @@ export default {
 @import '../../assets/variables';
 
 .wrapper {
-	position: relative;
-	display: flex;
-	justify-content: center;
-	padding: 12px 0 12px;
+	padding: 12px 0;
 	min-height: 69px;
-
-	&--has-typing-indicator {
-		padding: 30px 0 12px;
-	}
 }
 
-.new-message {
-	width: 100%;
+.new-message-form {
+	align-items: flex-end;
 	display: flex;
-	justify-content: center;
-
-	&-form {
-		align-items: flex-end;
-		display: flex;
-		position: relative;
-		flex: 0 1 700px;
-		margin: 0 4px;
-
-		&__emoji-picker {
-			position: absolute;
-			bottom: 1px;
-			z-index: 1;
-		}
-
-		&__input {
-			flex-grow: 1;
-			overflow: hidden;
-			position: relative;
+	position: relative;
+	max-width: 700px;
+	margin: 0 auto;
+
+	// Override NcRichContenteditable styles
+	& &__richContenteditable {
+		border: 1px solid var(--color-border-dark);
+		border-radius: calc(var(--default-clickable-area) / 2);
+		padding: 8px 16px 8px 44px;
+		max-height: 180px;
+
+		&:hover,
+		&:focus,
+		&:active {
+			border: 1px solid var(--color-primary-element);
 		}
+	}
 
-		// Override NcRichContenteditable styles
-		& &__richContenteditable {
-			border: 1px solid var(--color-border-dark);
-			border-radius: calc(var(--default-clickable-area) / 2);
-			padding: 8px 16px 8px 44px;
-			max-height: 180px;
-
-			&:hover,
-			&:focus,
-			&:active {
-				border: 1px solid var(--color-primary-element);
-			}
-		}
+	&__emoji-picker {
```

Comment:
![IMG_20230623_194627.jpg](https://github.com/nextcloud/spreed/assets/93392545/f0aa9d65-027f-4cab-ba44-8718778cab39)

Sorry, this unchanged styles, moved to another position, distract me. Can we try to minimize amount of changes in this PR. It could affect us later when we backport something to older versions